test/parallel/test-http-many-ended-pipelines.js

```diff
@@ -16,7 +16,13 @@ var numRequests = 20;
 var done = 0;
 
 var server = http.createServer(function(req, res) {
-  res.end('ok');
+
+  res.end('ok', common.mustCall(function() {}));
+
+  // We *might* get a socket already closed error here, which
+  // occurs when the socket was destroyed before we finished
+  // writing our data.
+  res.on('error', function() {});
```

Comment:
As it is we simply ignore everything right? As the comment says, if we check if the error is actually socket closed error, wouldn't it be better?